### ISSUE TITLE
groupWith?

### ISSUE THREAD

**Issue by AlexMouton** [purescript/purescript-arrays#135]
i see sortBy/sortWith, but only groupBy.
Minor convenience but it would be nicely parallel, and this missing one is often what I want:  Theres no new logic, I just need to dive into some structure.

**Comment by JordanMartinez** [comment]
Pretty sure we can close this now. `groupWith` was added recently under the name `groupAllBy`

**Comment by hdgarrood** [comment]
I don't think so; `groupWith` would be something like `forall a b. Ord b => (a -> b) -> Array a -> Array (NonEmptyArray a)`. It would be a shorthand for `\f -> groupBy (compare `on` f)` or something similar.
